webpack.config.js

```diff
@@ -0,0 +1,61 @@
+const path = require("path");
```

Comment:
`path` is not installed via package.json.

Reply:
indeed, but I am not sure how it was removed in the first place. However, re-added here [977b4ab](https://github.com/demos-europe/demosplan-ui/pull/12/commits/977b4abe8329ba499056a8d1a654b91edb1af98d)